pkg/render/monitor/monitor.go

```diff
@@ -138,6 +208,55 @@ func (mc *monitorComponent) Objects() ([]client.Object, []client.Object) {
 	return toCreate, toDelete
 }
 
+func (mc *monitorComponent) clusterRole() client.Object {
```

Comment:
Is it possible to reduce ClusterRole/ClusterRoleBinding to namespaced Role/RoleBinding?

Reply:
All three of the permissions below are cluster scoped, so they cannot be managed on a per-ns basis. That being said, I think I might be able to remove the authenticationreviews, so I will explore that now.